ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/query-src/dropdown.query-src.less

```diff
@@ -3,7 +3,7 @@
   display: block;
   white-space: nowrap;
   overflow: hidden;
-  padding: @minimumSpacing;
+  padding: 1.5rem;
```

Comment:
Is there a theme value that can be used, or added to ensure these all stay the same?